lib/util/Components.js

```diff
@@ -303,6 +303,16 @@ function componentRule(rule, context) {
       return calledOnReact;
     },
 
+    isReturningArrayMap(node) {
```

Comment:
i'm concerned about this approach; there's other things that have a `.map` method. How can we know this is an array?